### PR TITLE
Update Auth snippets to use Bootstrap 4

### PR DESCRIPTION
**Pull request by bjorntheart**
https://github.com/MasoniteFramework/core/issues/368 this PR attempts to replace Bootstrap 3 with Bootstrap 4 for `auth snippets`. It also takes into account @vaibhavmule's PR https://github.com/MasoniteFramework/core/pull/367 changes from `username` to `email`

Register
screen shot 2018-09-26 at 11 05 07
Dashboard

### CODE REVIEW

File: masonite/snippets/auth/templates/auth/register.html [hunk 1]
```diff
@@ -1,32 +1,35 @@
...
-            </div>
-
-            <div class="form-group">
-                <label for="">Password</label>
-                <input type="password" name="password" class="form-control">
-            </div>
-
-            <div class="form-group text-center">
-                <button type="submit" class="btn btn-success">Register</button>
+<div class="row my-5">
```
Comment (vaibhavmule) [inline masonite/snippets/auth/templates/auth/register.html]: What is `my-5` ?

Comment (bjorntheart) [reply masonite/snippets/auth/templates/auth/register.html]: https://getbootstrap.com/docs/4.1/utilities/spacing/#notation

Comment (vaibhavmule) [reply masonite/snippets/auth/templates/auth/register.html]: Wow, this is  interesting, I haven't came across this.

Good one.

File: masonite/snippets/auth/templates/auth/base.html [hunk 2]
```diff
@@ -6,41 +6,48 @@
...
     <title>{{ app.name }}</title>
-    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
-        crossorigin="anonymous">
+    <link rel="stylesheet"
+          href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+          integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
+          crossorigin="anonymous">
 </head>
+<style type="text/css">
+    html, body, input {
```
Comment (vaibhavmule) [inline masonite/snippets/auth/templates/auth/base.html]: You can just remove `input` from here as `input` tag will inherit the font size form `html` and `body`.

**Comment by josephmancuso** [comment]
@mitchdennett does bs4 do that hamburger menu still?

screen shot 2018-09-26 at 6 01 14 pm

**Comment by mitchdennett** [comment]
Did you mean @bjorntheart . But i believe it does

**Comment by josephmancuso** [comment]
@bjorntheart woops :)

**Comment by bjorntheart** [comment]
@josephmancuso I'll add the 🍔 menu

File: masonite/snippets/auth/templates/auth/base.html (hunk 2 above)
Comment (bjorntheart) [reply masonite/snippets/auth/templates/auth/base.html]: I made the necessary changes ;-)

File: masonite/snippets/auth/templates/auth/base.html [hunk 3]
```diff
@@ -37,9 +44,19 @@
...
-        {% block content %}{% endblock %}
     </div>
+</nav>
+<div class="container">
+    {% block content %}{% endblock %}
+</div>
+<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
+        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
+        crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
```
Comment (vaibhavmule) [inline masonite/snippets/auth/templates/auth/base.html]: I'm not sure if it needs to be added for Hamburger menu?

**Pull request merged**